added setBound method to Forest

ThopClayton opened the pull request:
Added the variables for min and max X/Z values. 

Added method setBound which will determine those values.

Added a line to main with an example call of the function (can be removed)

ThopClayton committed:
added setBound method to Forest

Noah-Schoonover reviewed (changes requested):
The method name `setBound()` sounds like the user is setting the bound, which might be a function we will create in the future. Will you consider changing the name to something like `calculateBounds()` or `updateBounds()` to better fit the task it's performing?

Also, this method should be called internally by the Forest class (after any operation that will change the forest, e.g. after import or random generation). It should not be called by the user in the main function. As a matter of fact, we should make the method `private`.

Noah-Schoonover committed:
Changed setBound() to updateBounds()

Noah-Schoonover committed:
Merge branch 'main' into setBound

Noah-Schoonover committed:
Other updates to Forest

Added constructor for reading from file, initialized bounds,
moved updateBounds() to be called within Forest
Added some exception handling
